### PR TITLE
Applied changes regarding to language

### PR DESCRIPTION
**Pull request by tjerman**

### CODE REVIEW

File: src/modules/end-user-guide/pages/crm/accounts.adoc [hunk 1]
```diff
@@ -32,26 +32,32 @@ You can create new contracts for the account by pressing the "Create new Contrac
 ====
 You can create new cases for the account by pressing the "Create new Case from this account" button.
 ====
+//Nika's note: if possible unite the tips inder 1 'TIPS' section. My attempt is bellow, but I couldnt figure out if I can make 'TIP' plural.
 
+[TIP]
+====
+* You can create new contracts for the account by pressing the "Create new Contract from this account" button.
+* You can create new cases for the account by pressing the "Create new Case from this account" button.
+====
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/accounts.adoc]: This is fine and valid; remove your comment and delete the duplicated tip blocks

File: src/modules/end-user-guide/pages/crm/accounts.adoc [hunk 2]
```diff
@@ -32,26 +32,32 @@ You can create new contracts for the account by pressing the "Create new Contrac
...
 
-== Add a new account manually
+== Manual addition of a new account
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/accounts.adoc]: I feel like this version is a bit too verbose

File: src/modules/end-user-guide/pages/crm/campaigns.adoc [hunk 3]
```diff
@@ -5,14 +5,15 @@ include::ROOT:partial$variables.adoc[][]
...
-A campaign can be anything from a simple newsletter, seminar, conference, or any other type of marketing initiative.
+A campaign can be anything; a simple newsletter, seminar, conference, or any other type of marketing initiative.
 
-Leads, contacts, accounts and opportunities are related to campaigns, as *campaign members*.
-The campaign - opportunity relationship allows you to calculate values such as the ROI (Return Of Investment) based on the campaign cost and expected revenue.
+Leads, contacts, accounts and opportunities are related to campaigns as *campaign members*.
+The campaign - opportunity relationship allows you to calculate values such as the ROI (Return of Investment) based on the campaign cost and the expected revenue.
 
 To calculate the ROI, you need to insert the cost of the campaign.
+//Nika's note: insert where? - clarity purposes
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/campaigns.adoc]: Can you just comment out the line in question (do the `//` before the line)

File: src/modules/end-user-guide/pages/crm/cases.adoc [hunk 4]
```diff
@@ -13,32 +13,33 @@ The CRM and the xref:service-solution/index.adoc[Service Solution] do not share
 ====
 
 Cases can be linked to accounts, contacts products or even to other cases.
+//Nika's note: contacts products? The products of contacts -> contact's, if it is meant as a plural, remove the "s" in contacts
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/cases.adoc]: contacts, products; missing `,`

File: src/modules/end-user-guide/pages/crm/accounts.adoc (hunk 2 above)
Comment (tjerman) [reply src/modules/end-user-guide/pages/crm/accounts.adoc]: The same applies to all of the sub titles

File: src/modules/end-user-guide/pages/crm/leads.adoc [hunk 5]
```diff
@@ -2,20 +2,20 @@ include::ROOT:partial$variables.adoc[][]
 
 = Leads
 
-The Leads page is accessible by clicking on the menu:Leads[] in the main navigation bar on the top of the screen.
+The Leads page is accessible by clicking on the menu:Leads[] section in the main navigation bar on the top of the screen.
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/leads.adoc:5]: All of the sections start with a variation of this; should all of them be changed?

File: src/modules/end-user-guide/pages/crm/leads.adoc [hunk 6]
```diff
@@ -43,28 +43,54 @@ This allows you to see, directly in the lead record, a list of latest tweets fro
 You can create notes, events and tasks for a lead.
 ====
 
+//Nika's note: I'd add the tips under a joined section, having four seperate sections named 'TIP' is avoidable (hopefully?). My attempts are below. Couldn't figure out how to make TIP plural (add the final s) in the 1st attempt.
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/leads.adoc]: The `[TIP]` thing makes a nice box around the text (see https://docs.cortezaproject.org/corteza-docs/2021.3/end-user-guide/crm/index.html#_add_a_new_lead for example).
You can't change the text (at least to my knowledge). Please just ignore this typo; I'll see if I can fix it later on (the same applies for all similar bits)

File: src/modules/end-user-guide/pages/crm/leads.adoc [hunk 7]
```diff
@@ -43,28 +43,54 @@ This allows you to see, directly in the lead record, a list of latest tweets fro
...
+==== TIPS
+* The rating field allows you to see which leads are qualified as good leads.
+* Try to retrieve as much relevant lead data as possible.
+It helps create interesting charts, such as leads per country or leads per industry.
+* Fill in the lead's personal or company twitter profile.
+This allows you to see, directly in the lead record, a list of latest tweets from that profile.
+* You can create notes, events and tasks for a lead.
+//Nika's note: having it like this lessens the consistency throughout the documents, so it is not optimal.
+
+
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/leads.adoc]: Remove this subsection in favour of the above TIP block; also remove all of the extra newlines (there should be only one above and below the TIP block.

File: src/modules/end-user-guide/pages/crm/leads.adoc [hunk 8]
```diff
@@ -43,28 +43,54 @@ This allows you to see, directly in the lead record, a list of latest tweets fro
...
+== The conversion of a lead
 
 A lead should be converted when there is a sales opportunity.
 
 [NOTE]
 ====
-When a lead is converted, the generated Contact is set as the primary contact for the account.
+When a lead is converted, the generated contact is set as the primary contact for the account.
 ====
+//Nika's note: any particular need for the 'contact' (1st) to be capitalised?
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/leads.adoc]: No

File: src/modules/end-user-guide/pages/crm/leads.adoc [hunk 9]
```diff
@@ -43,28 +43,54 @@ This allows you to see, directly in the lead record, a list of latest tweets fro
...
 
-If a lead *shows commercial interest*, such as a request for pricing and the lead's *location matches the region your business targets*, this lead could be considered a *5 star lead*.
+If a lead *shows commercial interest*, such as a request for pricing, and the lead's *location matches the region your business targets*, this lead could be considered a *5 star lead*.
 
-If a lead *performed some additional interactions*, such as going through a demo, they could be considered a *3 star lead*.
+If a lead *performs some additional interactions*, such as going through a demo, they could be considered a *3 star lead*.
 
-If a lead *contacted* you regarding, but it's clear that there is no explicit business intention, it can be considered a *0 star* or a *1 star* lead.
+If a lead *contacts* you regarding, but it's clear that there is no explicit business intention, it can be considered a *0 star* or a *1 star* lead.
+// Nika's note: regarding what? regarding your business, product...?
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/leads.adoc]: Either; something that may result in us doing business with the lead

File: src/modules/end-user-guide/pages/crm/quotes.adoc [hunk 10]
```diff
@@ -10,25 +10,25 @@ A quote allows you to complete the payment and thus charge the customer.
 
 [IMPORTANT]
 ====
-{PRODUCT_NAME} does not implement a payment mechanism so you need to use an external provider.
+{PRODUCT_NAME} does not implement a payment mechanism. You should use an external provider.
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/quotes.adoc]: One sentence per line!

```
{PRODUCT_NAME} does not implement a payment mechanism.
You should use an external provider.
```

This makes it easier for us to see what changed.

File: src/modules/end-user-guide/pages/crm/tasks.adoc [hunk 11]
```diff
@@ -6,50 +6,52 @@ The Tasks page is accessible by clicking on the menu:Tasks[] in the main navigat
 
 Tasks allow you to keep track of outstanding tasks that need to be handled and are displayed in the list or on the calendar on the homepage.
 Tasks can refer to leads, accounts, contacts, opportunities, quotes, or contracts.
+//Nika's note: outstanding seems inappropriate, try tasks that are out of the ordinary (if that is the meaning), substitute the word for sth else or leave the premodification out entirely
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/tasks.adoc]: Outstanding as in not yet resolved.
Feel free to omit.

File: src/modules/end-user-guide/pages/crm/tasks.adoc [hunk 12]
```diff
@@ -6,50 +6,52 @@ The Tasks page is accessible by clicking on the menu:Tasks[] in the main navigat
...
 . click on the btn:[+ Add] button.
 
-== Add a new task for a contract
+== The addition of a new task for a contract
 
 . Navigate to the menu:Contracts[select a contract],
-. scroll down to the "tasks" page block
+. scroll down to the "tasks" page block,
 . click on the btn:[+ Add] button.
+
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/crm/tasks.adoc]: Remove the extra new line; only one at the end of a file

File: src/modules/end-user-guide/pages/service-solution/cases.adoc [hunk 13]
```diff
@@ -7,54 +7,55 @@ The Cases page is accessible by clicking on the menu:Cases[] in the main navigat
...
 Each case has an automatically generated case number, which starts with "ticket#" followed by a number.
 An example case number is `ticket#20191025094353420`.
 
 [NOTE]
 ====
-The ticket number provides a unique identification that is used within the system.
+The ticket number provides a unique identification which is used within the system.
 To make the ticket understandable, provide a good subject.
 ====
+//Nika's note: provide a further explanation of what a good subject is. ->  To make the ticket understandable, provide a subject that is/includes/allows the program (you) to ...
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/service-solution/cases.adoc]: You should get a general idea of what the ticket is about from the subject alone. Such as "Client xyz is unable to access the internet with the router abc".

If you can't think of something better it as is along with your comment and we'll change it later.

File: src/modules/end-user-guide/pages/service-solution/cases.adoc [hunk 14]
```diff
@@ -7,54 +7,55 @@ The Cases page is accessible by clicking on the menu:Cases[] in the main navigat
...
 
 [TIP]
 ====
 Make sure that the subject is short, clear and straight to the point.
 ====
+//maybe an addition of an example could be nice
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/service-solution/cases.adoc]: You can use the example in my previous comment (or a variation of it).
Feel free to merge this tip block with the above note block.

File: src/modules/end-user-guide/pages/service-solution/entitlements.adoc [hunk 15]
```diff
@@ -7,18 +7,19 @@ The Entitlements page is accessible by clicking on the menu:Accounts[Entitlement
 Entitlements are types of customer support, such as *phone support* or *email support*.
 You can set up entitlements to help support agents determine whether a customer is eligible for support or not.
 
-Entitlements are related to an account and a product, and they are eligible for a fixed amount of time (defined by the start and end date).
+Entitlements are related to an account and a product. They are eligible for a fixed amount of time (defined by the start and end date).
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/service-solution/entitlements.adoc]: I sentence per line

File: src/modules/end-user-guide/pages/service-solution/entitlements.adoc [hunk 16]
```diff
@@ -7,18 +7,19 @@ The Entitlements page is accessible by clicking on the menu:Accounts[Entitlement
...
+//Nika's note: eligible doesn't fit here meaningwise, try to substitute the word. I'd suggest availible/accesible, but it depends on what you are really trying to say.
```
Comment (tjerman) [inline src/modules/end-user-guide/pages/service-solution/entitlements.adoc]: Eligible as in "I bought this, so I have access to some support/upgrade/replacement provided by the seller"

**Review by tjerman — COMMENTED** [review]
@nikabrlec please look into this.
In addition to the comments, each file has `include::ROOT:partial$variables.adoc[][]` in the first line. It should be `include::ROOT:partial$variables.adoc[]`; can you please fix this for me?

File: src/modules/end-user-guide/pages/crm/accounts.adoc (hunk 1 above)
Comment (nikabrlec) [reply src/modules/end-user-guide/pages/crm/accounts.adoc]: I remove the original or my attempt?

Comment (tjerman) [reply src/modules/end-user-guide/pages/crm/accounts.adoc]: Remove the original and keep your attempt

File: src/modules/end-user-guide/pages/crm/campaigns.adoc (hunk 3 above)
Comment (nikabrlec) [reply src/modules/end-user-guide/pages/crm/campaigns.adoc]: I clarified my comment and added the //.

File: src/modules/end-user-guide/pages/crm/leads.adoc (hunk 5 above)
Comment (nikabrlec) [reply src/modules/end-user-guide/pages/crm/leads.adoc:5]: Added everywhere.

File: src/modules/end-user-guide/pages/crm/leads.adoc (hunk 6 above)
Comment (nikabrlec) [reply src/modules/end-user-guide/pages/crm/leads.adoc]: I removed the original (as I did in 'accounts') and my second attempt.

**Comment by tjerman** [comment]
Merged

**Pull request closed without merging**